src/FacebookApi.cpp

```diff
@@ -21,9 +21,11 @@
 
 #include "FacebookApi.h"
 
-FacebookApi::FacebookApi(Client &client, String accessToken)	{
+FacebookApi::FacebookApi(Client &client, String appId, String appSecret, String accessToken)	{
```

Comment:
This is a breaking change to the existing library. If you moved them to the end of constructor you could default them to a blank string which would maintain backward compatibility.
`FacebookApi(Client &client, String accessToken, String appId = "", String appSecret = "")`